### ISSUE TITLE
Since 1.3.0: date not correctly validated

### ISSUE THREAD

**Issue by itineric** [networknt/json-schema-validator#938]
Consider the following schema:

```
{
  "type": "object",
  "properties":
  {
    "creation_date":
    {
      "type": "string",
      "format": "date-time"
    }
  }
}
```

The following JSON should not be valid but it is:
```
{
  "creation_date": "not a date"
}
```

This was working with version 1.2.0

**Comment by justin-tay** [comment]
In version 1.3.0 the `format` keyword by default complies with the specification that it should be an annotation since 2019-09 and configuration needs to be set to always generate an assertion or the required vocabulary needs to be in the meta schema.

https://json-schema.org/draft/2020-12/json-schema-validation#name-vocabularies-for-semantic-c

**Comment by rmuller** [comment]
Should not this be closed? Because as of 1.3.2 this works as expected (= according to the spec and documentation of this library)

**Comment by stevehu** [comment]
@rmuller Thanks a lot for your confirmation.
